protocols/virtualdefund/virtualdefund.go

```diff
@@ -304,6 +348,19 @@ func (o *Objective) Crank(secretKey *[]byte) (protocols.Objective, protocols.Sid
 		return &updated, sideEffects, WaitingForNothing, protocols.ErrNotApproved
 	}
 
+	if !updated.VoucherSent {
+
+		myAddress := updated.VFixed.Participants[updated.MyRole]
+		vMsgs := protocols.CreateVoucherMessage(*updated.Vouchers[o.MyRole], myAddress, updated.Id(), updated.VFixed.Participants...)
+
+		sideEffects.MessagesToSend = append(sideEffects.MessagesToSend, vMsgs...)
+		updated.VoucherSent = true
+	}
+
+	if !updated.haveAllVouchers() {
```

Comment:
We shouldn't have to wait for everyone's voucher, we just need alice to send her latest voucher.

Reply:
Updated in [c21c480](https://github.com/statechannels/go-nitro/pull/826/commits/c21c480acfbf7415242089764e5cc528525fefef). Now only Alice sends her voucher and we only store one voucher on the objective.